### ISSUE TITLE
Rename (or add) repository for homebrew

### ISSUE THREAD

**Issue by ryanlevy** [ALPSCore/packaging#3]
I'd like to suggest making a homebrew tap repository for all `ALPSCore` codes 
The full explanation is [here](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md), but in short: if we make a repository named `homebrew-tap` then a user simply has to use `brew tap alpscore/tap` and then they'll have all the formulas that we maintain, without waiting for pull requests. 

For example, see [my maxent tap](https://github.com/ryanlevy/homebrew-tap).

**Comment by aeantipov** [comment]
IMHO it is a good idea. Yet having stuff in homebrew-science has a perk of being automatically tested for all current Mac OS versions. So I'd say, it's better to leave alpscore itself where it is, while having a tap for all new codes.

**Comment by galexv** [comment]
Since [homebrew/science got discontinued](https://brew.sh/2018/01/19/homebrew-1.5.0/), we indeed need to make a separate tap for ALPSCore.

**Comment by mwallerb** [comment]
Bumped by @ryanlevy in an e-mail to me.

**Comment by egull** [comment]
Good point. @galexv, can you prioritize this once our Jenkins installation is done?

**Comment by galexv** [comment]
Yep. Self-assigned now. :)
